assets/js/chatbot-chatgpt.js

```diff
@@ -240,7 +243,17 @@ jQuery(document).ready(function ($) {
             scrollToBottom();
         }
     }
-
+    // Add the toggleChatOption() function - Ver 0.0.1
+    function toggleChatOption() {

```

Comment:
@NEVO-SAY-NEVO is it possible to close the option when you click outside the pop up?